### DISCUSSION TITLE
IPF color key

### DISCUSSION THREAD

**Discussion by jaywai412** [mtex-toolbox/mtex#709] [Ask Anything]
Hello all,

I imported osc files and plotted IPF colormaps.
In the manual, ipfTSLKey is said to be the function for using the default color key in OIM software.
But, I can reproduce the color maps that I had received from our operator when using ipfHSVKey function rather than ipfTSLKey.
Using ipfTSLKey function, I can get similar maps, but some of grains with close orientations are not discernible.  
My understanding on the color key function for reproducing maps generated by OIM software is wrong?

Comparing the color key generated by ipfHSVKey function (left) and the color key in OIM software (right), they also look slightly different as seen in the attached file. Can I use the same color key as the one shown on the right in MTEX? It seems different projection schemes are used in the two cases.

Thank you.

![colorkey](https://user-images.githubusercontent.com/73686295/104767891-40536580-57b0-11eb-8ef7-64420275f8a8.png)

**Comment by kilir** [comment]
Hi,
the possibility that you can get unsatisfying results with the TSL color key was the reason to have nicer color key in mtex (Orientations - perfectly colored, G. Nolze, R. Hielscher: Journal of Applied Crystallography, 49, 1786-1802, 2016, http://dx.doi.org/10.1107/S1600576716012942).
Recently OIM implemented actually the mtex color keys (https://edaxblog.com/2020/01/07/colorful-language/) because of their advantages. If you really want to mimick the old OIM key, you may have to design it yourself.

Cheers,
Rüdiger
